lms/models/solutions.py

```diff
@@ -195,7 +195,7 @@ def get_files_tree(files: Iterable[SolutionFile]) -> List[Dict[str, Any]]:
             'id': file.id,  # type: ignore
             'fullpath': file.path,
             'path': os.path.split(file.path.strip('/'))[1],  # type: ignore
-            'indent': file.path.strip('/').count('/'),  # type: ignore
+            'indent': file.path.strip('/').count('/') + 1,  # type: ignore
```

Comment:
Why?

Reply:
No it doesn't. If we wouldn't do it there will be a collision with the folders/files icons in the first hierarchy with the comments number.
Another option is to add `1` to the count in the html file.